src/qnngds/devices/nanowire.py

```diff
@@ -0,0 +1,35 @@
+from phidl import Device
+
+import phidl.geometry as pg
+from typing import Tuple, Optional
+
+
+def spot(
```

Comment:
not sure if `spot` is the best name, but we can discuss.

Also I think it might be nice to have it take a desired number of squares and add a straight in between the two tapers if the number of squares requested is large enough. I can put that in a separate issue. This is fine as is.

Reply:
totally agree with you, I had no idea how to name it but as the function is right now it is kind of a "hot-spot". Very good idea to upgrade the function to allow for straight line, I agree that this should go in another issue though.
Created #41